docs/sphinx/versions.rst

```diff
@@ -1,6 +0,0 @@
-************************
```

Comment:
Why was this removed? This file is needed for the docs versioning to work correctly.